frontend/vue/components/SignIn/SignIn.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <section class="sign-in">
+    <div class="sign-in__content">
+      <div class="sign-in__content-section sign-in__content-section__hero">
```

Comment:
I think we could remove `-section` from this class...  and stick w/ `sign-in__content__hero`, and in the next line `sign-in__content__hero-title`